Rakefile

```diff
@@ -167,3 +177,23 @@ namespace :pkg do
     `bzip2 -9 #{PKGDIR}/foreman-installer-#{version}.tar`
   end
 end
+
+
+namespace :config do
+  task :migrate do
```

Comment:
Isn't this the same as `bin/foreman-installer --migrations-only`?

(I'm hesitant to add code using Kafo internals if unnecessary)